src/crud/workspace.py

```
@@ -0,0 +1,97 @@
+            metadata=workspace.metadata or {},  # Provide empty dict if metadata is None
+        ),
+    )
+
+    if workspace.metadata is not None:
+        honcho_workspace.h_metadata = workspace.metadata
+
+    if workspace.configuration is not None:
+        honcho_workspace.configuration = workspace.configuration
+
+    await db.commit()
+    logger.info(f"Workspace with id {honcho_workspace.id} updated successfully")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)